pass-download-service integration tests fail

Running integration tests fails with:
```
docker-compose up -d
go test -tags=integration ./...
```

The result is:
```
http://localhost:8080/fcrepo/rest/files/c3/c0/28/eb/c3c028eb-4d3a-4938-9ff2-8abf56d9d8d8
--- FAIL: TestIntegration (9.61s)
    integration_test.go:48: Didn't get expected content length for pdf file
```

Interestingly I can download that pdf from fcrepo.

--- a/integration_test.go
+++ b/integration_test.go
@@ -44,7 +44,7 @@ func TestIntegration(t *testing.T) {
 		t.Fatalf("Got wrong content type for PDF!")
 	}
 
-	if resp.Header.Get("Content-Length") != "1024793" {
+	if resp.Header.Get("Content-Length") != "1035596" {
 		t.Fatalf("Didn't get expected content length for pdf file")
 	}
 }

